docs/projects/slopi-communication.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Searchable Linkable Open Public Indexed (SLOPI) Communication
+nav_order: 3
+parent: Projects
```

Comment:
Did you change the nav order so it comes next in line? Off the top of my head we have 3 projects, so maybe this would be 4?

To test locally (in case you haven't).

```bash
$ bundle exec jekyll serve
```

Reply:
Sure, I can change this to 4. However, please note that as of f946538 on master there are already two with a nav_order of 3:

```
docs/projects/funding.md:nav_order: 3
docs/projects/human-menu.md:nav_order: 3
```

So maybe I should use 5?

Reply:
Yeah and please change one of those to 4 because apparently I can't count, heh :)